src/homepageExperience/components/HomepageExperience.scss

```diff
@@ -156,3 +159,39 @@ $pool-fill: #00a3ff;
   align-items: center;
   text-transform: uppercase;
 }
+
+.homepage-wizard-next-steps {
```

Comment:
in a follow up ticket (not this one), can you style up the `HomePageContainer` similarly to the way this page is styled?